WordPress/WordPressTest/ReaderSelectInterestsCoordinatorTests.swift

```diff
@@ -0,0 +1,169 @@
+import XCTest
+@testable import WordPress
+
+// MARK: - MockFollowedInterestsService
+class MockFollowedInterestsService: ReaderFollowedInterestsService {
+    var success = true
+    var fetchSuccessExpectation: XCTestExpectation?
+    var fetchFailureExpectation: XCTestExpectation?
+
+    private let failureError = NSError(domain: "org.wordpress.reader-tests", code: 1, userInfo: nil)
+
+    private var testContextManager: CoreDataStack?
+    private var context: NSManagedObjectContext?
+
+    init(populateItems: Bool) {
+
+        testContextManager = TestContextManager.sharedInstance()
+        context = testContextManager?.mainContext
+
+        // Don't populate the objects
+        guard populateItems else {
+            return
+        }
+
+        populateTestItems()
+    }
+
+    // MARK: - Fetch Methods
+    func fetchFollowedInterestsLocally(completion: @escaping ([ReaderTagTopic]?) -> Void) {
+        guard self.success else {
+            fetchFailureExpectation?.fulfill()
+
+            completion(nil)
+            return
+        }
+
+        let interests = followedInterests()
+
+        completion(interests)
+        fetchSuccessExpectation?.fulfill()
+    }
+
+    func fetchFollowedInterestsRemotely(completion: @escaping ([ReaderTagTopic]?) -> Void) {
+        self.fetchFollowedInterestsLocally(completion: completion)
+    }
+
+    // MARK: - Private: Helpers
+    private func populateTestItems() {
+        guard let context = context else {
+            XCTFail("Context is nil")
+            return
+        }
+
+        let interest = NSEntityDescription.insertNewObject(forEntityName: "ReaderTagTopic", into: context) as! ReaderTagTopic
+        interest.path = "/tags/interest"
+        interest.title = "interest"
+        interest.type = ReaderTagTopic.TopicType
+        interest.following = true
+
+        do {
+            try context.save()
+        } catch let error as NSError {
+            XCTAssertNil(error, "Error seeding topics")
+        }
+    }
+
+    private func followedInterestsFetchRequest() -> NSFetchRequest<ReaderTagTopic> {
+        let entityName = "ReaderTagTopic"
+        let predicate = NSPredicate(format: "following = YES")
+        let fetchRequest = NSFetchRequest<ReaderTagTopic>(entityName: entityName)
+        fetchRequest.predicate = predicate
+
+        return fetchRequest
+    }
+
+    private func followedInterests() -> [ReaderTagTopic]? {
+        let fetchRequest = followedInterestsFetchRequest()
+        do {
+            guard let interests = try context?.fetch(fetchRequest) else {
+                return nil
+            }
+
+            return interests
+        } catch {
+            XCTAssertNil(error, "Error fetching interests")
+
+            return nil
+        }
+    }
+}
+
+class ReaderSelectInterestsCoordinatorTests: XCTestCase {
+    func testShouldDisplay() {
```

Comment:
```suggestion
    func testShouldDisplayReturnsTrue() {
```

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/pull/14476/commits/f8df5f7bde0afb37349061e7a2f9e603a17f9ec7